app/views/transfers/dashboards/variant-1.html

```diff
@@ -140,25 +148,35 @@ <h2 class="govuk-heading-l">Filter</h2>
       </div>
     </form>
   </div>
-
-  <div class="govuk-grid-column-two-thirds">
+-->
+<div class="govuk-width-container"><br>
```

Comment:
We don't need another `width-container` (the one above should be deleted also) - there's one applied to the whole page already, so we can just add new `govuk-grid-row`s as required, otherwise a lot of spacing will be added where it isn't required

Reply:
Done.